index.html

```diff
@@ -1049,8 +1236,13 @@ <h3>Role Mapping Table</h3>
 								</td>
 								<td><p>Expose <code>ROLE_LANDMARK</code> and object attribute
 											<code>xml-roles:doc-prologue</code>. </p></td>
-								<td>AXRole: <code>AXGroup</code><br /> AXSubrole: <code>AXLandmarkRegion</code><br />
-									AXRoleDescription: <code>'region'</code>
+								<td>
+									<ul>
+										<li>AXRole: <code>AXGroup</code></li>
+										<li>AXSubrole: <code>AXLandmarkRegion</code></li>
+										<li>AXRoleDescription: <code>'region'</code></li>
+										<li>AXCustomContent:<code>{ label: "type", value: "prolog" }</code></li>
```

Comment:
[sic] could be prologue in the en-uk l10n

Reply:
Maybe an issue with this diff preview but it looks like this is modifying "doc-pagebreak" not "doc-preface" ??
ie.

> <td><p>Expose <code>ROLE_LANDMARK</code> and object attribute
> <code>xml-roles:doc-pagebreak</code>. </p></td>
```
+	<td>AXRole: <code>AXGroup</code><br /> AXSubrole: <code>AXLandmarkRegion</code><br />
+	<td>
+	<ul>
+		<li>AXRole: <code>AXGroup</code></li>
+		<li>AXSubrole: <code>AXLandmarkRegion</code></li>
+		<li>AXRoleDescription: <code>'region'</code></li>
+		<li>AXCustomContent:<code>{ label: "type", value: "preface" }</code></li>
+	</ul>

```

Reply:
I don't understand. You're commenting on the section covering `prologue` (not `pagebreak` or `preface`).

Reply:
Found it in a different section. For future reference, you can click on the line number to add a new comment rather than overloading an unrelated conversation.

Reply:
You can also click the button in the left of the blue collapse row (labeled "Expand All") to see additional diffs. That shows that I am editing the correct "preface" section, but the previous API seems so erroneously include "pagebreak". Will you file a new issue @clapierre? Good catch.

Reply:
Filed issue #16